src/components/OngoingIncidents/OngoingIncidents.js

```diff
@@ -1,10 +1,12 @@
 import React, { useEffect } from 'react'
 import { Link } from 'react-router-dom'
 import { useSelector, useDispatch } from 'react-redux'
+import { formatDate } from '../../utilities'
 import {
   getOngoingIncidents
 } from './OngoingIncidentsSlice'
 import './OngoingIncidents.css'
+import { ReactComponent as Fire } from '../../Icons/gripfire-brands.svg'
 
```

Comment:
Sounds like you're working on a way to make these icon imports dynamic for the different incident types. The icons look really cool and add nicely to the user experience.